packages/go_router/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+name: go_router
+description: A declarative router for Flutter based on Navigation 2 supporting
+  deep linking, data-driven routes and more
+version: 3.0.1
+repository: https://github.com/flutter/packages/tree/master/packages/go_router
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+go_router%22
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=1.17.0"
+
+dependencies:
+  collection: ^1.15.0
+  flutter:
+    sdk: flutter
+  flutter_web_plugins:
+    sdk: flutter
+  logging: ^1.0.0
+  path_to_regexp: ^0.4.0
```

Comment:
This is not a 1P package, so we should not depend on it. How hard would this be to remove?

Reply:
It can be done, but it may take some time. Is the rule in flutter/packages to only depend on first party packages ?

Reply:
Yes, unless there's an extremely compelling reason. And we definitely shouldn't be depending on a package that appears to only have a single maintainer, as that could easily leave us with a dependency on an abandoned package (which we would have to permanently fork if we ever really needed a fix, like a security issue or something similar to the NNBD migration).